### ISSUE TITLE
[Bug]: Unable to set quest_locked_icon

### ISSUE THREAD

**Issue by MetalKnight56** [FTBTeam/FTB-Mods-Issues#1351]
### Mod

FTB Quests

### Mod version

ftb-quests-forge-2001.4.8

### Forge / Fabric version

1.20.1 - 47.2.20

### Modpack & version

Custom

### What issue are you having?

Basically i'm unable to add a locked icon to locked quests, even tough i'm using the "quest_locked_icon" parameter. Im unsure if this is something to do with how i've written it or something, but i'm using a resourcepack that has this 
![image](https://github.com/user-attachments/assets/05efe83f-6f27-4d8e-a635-826efaa62a3e)

Inside of ftb_quests_theme.txt, so it should be working right? 

### Crashlogs

No crashlogs

### Steps to reproduce

Basically add this to any resourcepack in the following path: assets\ftbquests\
[ftb_quests_theme.txt](https://github.com/user-attachments/files/17237844/ftb_quests_theme.txt)

**Comment by desht** [comment]
There is no `quest_locked_icon` setting in the theme file, never has been. Where did you get the information that this exists?

**Comment by MetalKnight56** [comment]
> There is no `quest_locked_icon` setting in the theme file, never has been. Where did you get the information that this exists?

I actually just assumed there was as each icon was basically structured as "element_icon" and so I thought I could add an icon to that element by adding _icon...

Maybe it could be added as a new parameter in the theme file then, but this post should be re tagged as a suggestion instead then

**Comment by MetalKnight56** [comment]
Lmao, it's funny cuz i did answer and i knew this would happen, ig this is an ongoing occurance seen as how your issue settings are set...

**Comment by desht** [comment]
Yeah the bot isn't great at removing the stale label :frowning_face:  Although it seemed to work that time.

I will mark this as Feature Request, since it might be useful to have.

**Comment by manmaed** [comment]
Please disregard the previous reply from the stale bot, it's been a bad bot and sent for recycling.
